kolibri/logger/models.py

```diff
@@ -98,10 +101,16 @@ class ContentSessionLog(BaseLogModel):
     start_timestamp = DateTimeTzField()
     end_timestamp = DateTimeTzField(blank=True, null=True)
     time_spent = models.FloatField(help_text="(in seconds)", default=0.0, validators=[MinValueValidator(0)])
-    progress = models.FloatField(default=0, validators=[MinValueValidator(0)])
+    progress = models.FloatField(default=0, validators=[MinValueValidator(0), MaxValueValidator(1.0)])
     kind = models.CharField(max_length=200)
     extra_fields = JSONField(default={}, blank=True)
 
+    def save(self, *args, **kwargs):
+        if self.progress < 0 or self.progress > 1.01:
```

Comment:
why is this `self.progress > 1.01` and not `self.progress > 1.0`?

Reply:
+1

Or should I say... +0.01?